dependencies/current/trilinos/install.sh

```diff
@@ -99,8 +99,7 @@ $CMAKE_COMMAND \
     -D ParMETIS_INCLUDE_DIRS:PATH="/usr/include" \
     -D ParMETIS_LIBRARY_DIRS:PATH="/usr/lib/libparmetis.so.4.0.3" \
   -D TPL_ENABLE_UMFPACK:BOOL=ON \
-    -D UMFPACK_INCLUDE_DIRS:FILEPATH="$INSTALL_DIR/include" \
-    -D UMFPACK_LIBRARY_DIRS:FILEPATH="$INSTALL_DIR/lib" \
+    -D UMFPACK_INCLUDE_DIRS:FILEPATH="/usr/include/suitesparse" \
```

Comment:
Do we even need to point to this directory or can `Find_Package()` just do its thing on its own?